nilearn/plotting/find_cuts.py

```diff
@@ -64,6 +64,9 @@ def find_xyz_cut_coords(img, mask=None, activation_threshold=None):
     data = as_ndarray(data)
     my_map = data.copy()
     if mask is not None:
+        # check against empty mask
+        if mask.sum() == 0.:
+            raise ValueError("Provided mask is empty!")
```

Comment:
Maybe another option, possibly more useful for the end user, would be to raise this as a warning, and return the geometrical center of the mask.